Promise

the-nutritionist-that-code opened the pull request:
i am requesting a crate pull to make some changes and i will commit soon

the-nutritionist-that-code committed:
Create Product-list.html

i ceeated the product list

Commit:
the updated version

the updated version of product list
picture found and nav bar created

Commit:
nav bar and section 1

i have fully updated the nav bar and partially completed the section 1
having some problems but i would love to take a nap as  my eye lid is closing

Commit:
pictures upgraded

product created

Commit:
updated file but not final

corrections made except a little touch

Abdulmuiz00 commented:
it says here that you are merging to your main,  I think that you are to merge to your branch.

Commit:
Finally Done

i am submitting my final quota.
done with my responsiveness and the final project.

Commit:
FINALIZED

Commit:
updated

Commit:
finalize
